src/Sarif/CodeGenHints.json

```diff
@@ -731,6 +731,26 @@
       }
     }
   ],
+  "ResultProvenance.FirstDetectionTimeUtc": [
```

Comment:
>ResultProvenance [](start = 3, length = 16)

I see. So we just had inconsistently applied hints pointing to this converter for all UTC types in the schema.